frontend/src/components/widgets/BaseWidget/WidgetLabel.tsx

```diff
@@ -27,19 +28,29 @@ export interface LabelProps {
 
   // Used to specify whether widget disabled or enabled.
   disabled?: boolean | null
+
+  // Used to specify whether widget is visible or not.
+  labelVisibility?: LabelVisibilityOptions
 }
 
 export function WidgetLabel({
   label,
   children,
   disabled,
+  labelVisibility,
 }: LabelProps): React.ReactElement {
   if (label == null) {
     return <></>
   }
 
   return (
-    <StyledWidgetLabel disabled={disabled}>
+    // We manually disable StyeldWidgetLabel's aria-hidden,
+    // each widget should have its own aria-label and/or implement accessibility.
+    <StyledWidgetLabel
+      aria-hidden="true"
```

Comment:
Nit - this comment initially confused me: "We manually disable aria-hidden..." followed by `aria-hidden="true"`. 

I think maybe it's more accurate to say "we use `aria-hidden` to disable ARIA for StyleWidgetLabel, because each widget should..."